docs/F-Refinements/character-strings.md

```diff
@@ -298,7 +298,7 @@ The `scanf()` function will stop accepting input after the character `y` and sto
 <tr><td colspan="31" align="center">char<br/>name</td></tr>
 <tr><td>0</td>
     <td>1</td>
-    <td><strong>2</strong><td>
+    <td><strong>2</strong></td>
```

Comment:
Good catch.  Next time, do this in a separate PR, since this is unrelated to this fix.

Reply:
Will create an issue for those